docs/source/compatibility.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Compatibility
+description: Compatibility
+---
+
+import { Link } from 'gatsby';
+
+The Apollo Router uses Federation 2.x. However, Federation 2.x is [backward compatible with Federation 1.x](https://www.apollographql.com/docs/federation/federation-2/backward-compatibility/).
+
+As long as your graph is valid then your Federation 1 graph will work.
+
+Apollo Federation is an evolving project, and it will receive new features and bug fixes over time. If you need to run a particular version of Federation then the following table shows the version that the Router releases are compiled against.  
+
+<table class="field-table api-ref">
```

Comment:
Don't need these classes on this particular table (they're specifically for lists of function parameters or object fields)